OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -337,7 +343,11 @@ @implementation OHHTTPStubsProtocol
 
 + (BOOL)canInitWithRequest:(NSURLRequest *)request
 {
-    return ([OHHTTPStubs.sharedInstance firstStubPassingTestForRequest:request] != nil);
+    BOOL found = ([OHHTTPStubs.sharedInstance firstStubPassingTestForRequest:request] != nil);
+    if (!found && OHHTTPStubs.sharedInstance.onStubMissingBlock) {
+        OHHTTPStubs.sharedInstance.onStubMissingBlock(request);
```

Comment:
Did you check that this is only called once? I remember when I worked on `canInitWithRequest` a while ago that iOS called that method more than once to test a single request.
I have no idea why it would call it multiple times while calling it only one should be enough for the system, but in practice, at least some iOS versions ago when I worked on this part of the code, putting a breakpoint there made it be triggered multiple times even for only one request, so better check in case it's still the case (especially with old iOS version, as maybe they've improved that since recent updates hopefully)